src/ChecklistItem.js

```diff
@@ -0,0 +1,19 @@
+import '../node_modules/font-awesome/css/font-awesome.min.css';
+
+export function ChecklistItem(props) {
+    const inputTag = props.completed ?
+        <input key={props.id} type="checkbox" onInput={props.checkFunction} id={props.id}
```

Comment:
There's no difference between these two <input>s, so no need for ternary operator.